lib/views/widgets/dialog/dialog_utils.dart

```diff
@@ -0,0 +1,49 @@
+
+import 'package:flutter/material.dart';
+
+class DialogUtils {
+
+  static void showDialogBottomSheet(BuildContext context, Widget child) {
+    showModalBottomSheet(
+      context: context,
+      isScrollControlled: true,
+      shape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(16),
+          topRight: Radius.circular(16)),
+      ),
+      barrierColor: Colors.black38,
+      backgroundColor: Colors.white,
+      constraints: const BoxConstraints(maxWidth: 716),
+      builder: (_) {
+        return LayoutBuilder(
+            builder: (context, _) {
+              return AnimatedPadding(
+                  padding: EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
+                  duration: const Duration(milliseconds: 150),
+                  curve: Curves.easeOut,
+                  child: child
+              );
+            }
+        );
+      }
+    );
+  }
+
+  static void showDialogCenter(BuildContext context, Widget child) {
+    showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          contentPadding: EdgeInsets.zero,
+          shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(16)),
```

Comment:
I think we can take advantage of const keyword in here, but with a little longer code. 
`const RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(10)))`

Reply:
Oh, it's the same you. You can use two ways if you want.